entries-api/support.js

```diff
@@ -10,15 +10,16 @@ window.addEventListener('DOMContentLoaded', e => {
   header.innerText = document.title;
   document.body.appendChild(header);
   const elem = document.createElement('div');
-  elem.style = 'height: 50px; border: 1px dotted red;';
+  elem.style.cssText = 'height: 50px; border: 1px dotted red;';
   elem.innerHTML = 'Drop the <b>support/upload</b> directory here.</div>';
   document.body.appendChild(elem);
   elem.addEventListener('dragover', e => {
     e.preventDefault();
   });
   elem.addEventListener('drop', e => {
     e.preventDefault();
-    for (const item of e.dataTransfer.items) {
+    for (var i = 0; i < e.dataTransfer.items.length; ++i) {
```

Comment:
nit: this also doesn't have to be var, right?

Reply:
Made `const`.